doc/rst/tools/chpl-language-server/chpl-language-server.rst

```diff
@@ -77,7 +79,7 @@ syntax highlighting in Emacs):
 
   (with-eval-after-load 'eglot
     (add-to-list 'eglot-server-programs
-                 '(chpl-mode . ("chpl-language-server"))))
+                 '(chpl-mode . ("chpl-language-server", "--chplcheck"))))
```

Comment:
Sorry for not catching it earlier, but I think you can't use a `,` here, I was having trouble with that when making the linter command earlier.  I think this line should be `'(chpl-mode . ("chpl-language-server" "--chplcheck"))))` instead

Reply:
I have fixed this in https://github.com/chapel-lang/chapel/pull/25221